src/resource_clients/actor.ts

```diff
@@ -2,7 +2,7 @@ import type { AxiosRequestConfig } from 'axios';
 import ow from 'ow';
 
 import type { RUN_GENERAL_ACCESS } from '@apify/consts';
-import { ACT_JOB_STATUSES, META_ORIGINS } from '@apify/consts';
+import { ACT_JOB_STATUSES, ACTOR_PERMISSION_LEVEL, META_ORIGINS } from '@apify/consts';
```

Comment:
looks like the `ACTOR_PERMISSION_LEVEL` was introduced in `@apify/consts` v2.42.0, but we depend on ^v2.25.0 here, we should bump the lower bound to be sure

https://github.com/apify/apify-client-js/blob/master/package.json#L65

Reply:
Bumped :+1: